src/test/java/com/wikia/webdriver/testcases/globalnavigationbar/Navigating.java

```diff
@@ -10,18 +10,79 @@
 @Test(groups = {"globalnavigationbar", "globalnavigationbarNavigating"})
 public class Navigating extends NewTestTemplate {
 
-  private final String FANDOM_URL = "http://www.wikia.com/fandom";
-
-  @Execute(onWikia = "muppet")
   @Test(groups = {"fandomLogoClickOnEnCommunityOpensFandomWikia"})
   public void logoClickOnEnglishCommunityOpensFandom() {
-    HomePage homePage = new HomePage();
-    GlobalNavigation globalNav = homePage.getGlobalNavigation();
+    new HomePage()
+        .getGlobalNavigation()
+        .clickFandomLogo();
+
+    Assertion.assertEquals(driver.getCurrentUrl(), urlBuilder.getUrlForPage("fandom", "/"));
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Define a constant instead of duplicating this literal "fandom" 5 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1192)